src/adapt.hpp

```diff
@@ -99,6 +99,14 @@ class distributed_grid
     return plan_.at(rank);
   }
 
+  void
+  recreate_table(std::vector<int64_t> const &element_ids, int const max_level)
+  {
+    table_.recreate_from_elements(element_ids, max_level);
+
+    plan_ = std::move(get_plan(get_num_ranks(), table_));
```

Comment:
```suggestion
    plan_ = get_plan(get_num_ranks(), table_);
```

std::move should have no effect.